app/views/categories/index.html.erb

```diff
@@ -0,0 +1,32 @@
+<%= render 'header' %>
+<main class="categories">
+  <div class="credit-info">
+    <p class="gift-text">Total Amount</p>
+    <p class="gift-amt"><%= Deal.where(author_id: current_user.id).sum(:amount) %>
+    </p>
+  </div>
+  <div class="categories-container">
+    <div class="category-scroll">
+      <% if @categories.present? %>
+        <% @categories.each do |category| %>
+          <%= link_to user_category_path(current_user, category) do %>
+            <div class="category-container">
+              <%= image_tag url_for(category.icon.variant(resize: "100x100")) if category.icon.attached? %>
+              <div class="cat-name-date">
+                <h2 class="cat-name"><%= category.name %></h2>
+                <p class="cat-date"><%= category.created_at %></p>
+              </div>
+              <span class="cat-total-amt"><%= category.deals.sum(:amount) %></span>
+            </div>
+          <% end %>
+        <% end %>
```

Comment:
- [ ] I'm not able to see the uploaded icon of categories, They're broken. Please make sure to fix this and categories icons should display on the screen.

![image](https://github.com/ometman/budget-app/assets/83803414/79705567-935b-4b9d-ac5d-624c2524b861)